mscore/timeline.cpp

```diff
@@ -57,7 +57,7 @@ void MuseScore::showTimeline(bool visible)
             }
       connect(_timeline, SIGNAL(visibilityChanged(bool)), act, SLOT(setChecked(bool)));
       connect(_timeline, SIGNAL(closed(bool)), act, SLOT(setChecked(bool)));
-      _timeline->setVisible(visible);
+      reDisplayDockWidget(_timeline, visible); // _timeline->setVisible(visible);
```

Comment:
and here

Reply:
and fixed. Thanks for careful attention on this and helping me learn the ropes.